### PR TITLE
Unfreeze eventlet version

### PR DESCRIPTION
**Pull request by ryshah**
Kilo Swift needs version > 0.16.1 and reason that eventlet was set to 0.15.2 for Nova is not valid. Did a cloud deploy (from source) with latest eventlet and there were no issues

### CODE REVIEW

**Comment by ryshah** [comment]
Fix for issue #1205

**Comment by ryshah** [comment]
@j2sol @craigtracey Can we get this merged today?

File: roles/common/tasks/python.yml [hunk 1]
```diff
@@ -4,8 +4,8 @@
        version={{ item.rev }} accept_hostkey=True
   with_items: common.python_extra_packages
 
-- name: freeze eventlet to 0.15.2 for nova to work
-  pip: name=eventlet version=0.15.2
+- name: pip install eventelt
```
Comment (j2sol) [inline roles/common/tasks/python.yml]: This spelling is wrong. Fix it up and we'll merge without waiting for another test.

**Comment by j2sol** [comment]
Okay to merge once the spelling mistake is fixed.

**Comment by ryshah** [comment]
@j2sol Nice eye :+1: Fixed spelling

**Pull request merged**